studioShared/schemas/objects/listBlock.ts

```diff
@@ -0,0 +1,59 @@
+import { defineField } from "sanity";
+
+import { isInternationalizedString } from "studio/lib/interfaces/global";
+import { firstTranslation } from "studio/utils/i18n";
+const listBlock = defineField({
+  name: "listBlock",
+  title: "List Block",
+  description:
+    "This block could be used to add a list of skills, tools, methods etc. used for this project.",
+  type: "object",
+  fields: [
+    {
+      name: "description",
+      title: "Description",
+      description:
+        "Please provide a brief sentence describing the list. For example: The following skills were applied in this project:",
+      type: "internationalizedArrayString",
+    },
+    {
+      name: "list",
+      title: "List",
+      description:
+        "Add the items to be included in this list (e.g., skills, tools, methods).",
+      type: "array", // Make it an array
+      of: [
+        {
+          type: "object", // You need to define it as an object since it's a custom field
+          name: "internationalizedItem",
+          fields: [
+            {
+              name: "listItem",
+              title: "List Item",
```

Comment:
The titles in the edit dialog are not super clear:
<img width="680" alt="image" src="https://github.com/user-attachments/assets/e8eeab45-1d49-4fa1-9c66-91b534197b6d">

Here is one possible naming


```suggestion
          title: "List Item",
          name: "listItem",
          fields: [
            {
              name: "text",
              title: "Text",
```